app/build.gradle

```diff
@@ -30,6 +30,8 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.0.2'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.material:material:1.0.0'
+    implementation "com.google.android.gms:play-services-auth:17.0.0"
```

Comment:
If we add this dependency we are no longer showing auth without play services ... we're just showing Auth without the Firebase SDK.

I understand this is useful for Android Wear (which has Google Sign In but not Firebase Auth) but the main use cases for this repo are for devices without Play services entirely like Kindle Fire or Huawei